client/src/components/LoginSession/LoginSession.js

```diff
@@ -111,39 +52,37 @@ function LoginSession(props) {
     );
 
     return (
-      <div className={`login-session ${(loading.complete && !loading.failed) ? 'hidden' : ''}`}>
+      <div className={`login-session ${(props.complete && !props.failed) ? 'hidden' : ''}`}>
         <p>{props.UserAgent}</p>
         <p className="text-muted">
           {props.IPAddress}
           <span data-toggle="tooltip" data-placement="top" title={activityTooltip}>
             , {lastActiveStr}
           </span>
         </p>
-        {props.IsCurrent &&
         <p>
-          <strong className={'text-success'}>
-            {currentStr}
-          </strong>
+          {props.IsCurrent &&
+            <strong className="text-success">{currentStr}</strong>
+          }
+          {!props.IsCurrent && <Button
+            color="link"
+            className="login-session__logout"
+            onClick={() => attemptLogOut()}
```

Comment:
That's a bit more succinct. 

```suggestion
            onClick={attemptLogOut}
```